.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,133 @@
+# Because this library provides extension modules for macOS, but not for other
```

Comment:
No need to include that file. I will add it to master specifically for Python 2.7-3.5.

Reply:
I've reverted my changes to this file but kept it in the branch as a whole so that I don't have to mess about with git history.